### PR TITLE
#30 Unsuspend a user from failed logon attempts

### PR DESCRIPTION
**Pull request by AndrewCopeland**
#30 Unsuspend a user from failed logon attempts

### CODE REVIEW

**Comment by AndrewCopeland** [comment]
Having issues with the testing environment. I will inform you when a review is required.

**Comment by AndrewCopeland** [comment]
The v2 endpoint expects the `UserID` as a number rather than the username. I am thinking about using the old endpoint so its easier and more intuitive for the CLI user to just provide a username.

**Comment by infamousjoeg** [comment]
> The v2 endpoint expects the `UserID` as a number rather than the username. I am thinking about using the old endpoint so its easier and more intuitive for the CLI user to just provide a username.

I believe that using the older endpoint is okay in this case.  There weren't any _real_ changes otherwise between the two than that annoying change to UserIDs...

**Comment by AndrewCopeland** [comment]
Tested successfully via the API and CLI. You can review now

File: cmd/users.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			return
+		}
+
+		err = client.UnsuspendUser(Username)
+		if err != nil {
+			log.Fatalf("Failed to unsuspend user '%s'. %s", Username, err)
+			return
+		}
+
+		fmt.Printf("Successfully unsuspended user'%s'\n", Username)
```
Comment (infamousjoeg) [inline cmd/users.go:41]: Prints to STDOUT as `Successfully unsuspended user'cartman'` -- please add a space between `user'%s'`

**Review by infamousjoeg — CHANGES_REQUESTED** [review]

**Review by infamousjoeg — APPROVED** [review]

**Pull request merged**